services/cadip/config/cadip_sessions_stac_mapper.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Idem, do we need to add this file to rs-demo in https://github.com/RS-PYTHON/rs-demo/tree/main/local-mode/main/config and https://github.com/RS-PYTHON/rs-demo/blob/main/local-mode/main/scripts/rspy-pre-actions.sh#L17 ?

Reply:
No needed, since this is part of rs-server-cadip service.